question8.py

```diff
@@ -0,0 +1,7 @@
+numbers = [2, 5, 6, 1, 3, 8, 9, 10]
+
+for i in range(len(numbers)-1):
+    for j in range(len(numbers)-1):
+        if numbers[j] > numbers[j+1]:
+            numbers[j], numbers[j+1] = numbers[j+1], numbers[j]
```

Comment:
Can you tell me the sorting algorithm name you have used?

Reply:
its called bubble sort